DEPENDENCIES.md

```diff
@@ -0,0 +1,16 @@
+# OpenDAL Dependencies
+
+Please visit every package's `DEPENDENCIES.md` for a full list of dependencies.
+
+## Notes
+
+- `ring` has complex licenses that can't be detected by `cargo deny` correctly. We can use it as `ISC AND MIT AND OpenSSL`.
```

Comment:
Let's try to add a deny config for resolving this issue - https://github.com/briansmith/ring/issues/902#issuecomment-736976047

Reply:
DONE